.github/workflows/register-buildpack.yml

```diff
@@ -0,0 +1,260 @@
+              } catch (error) {
+                console.error(error)
+                process.exit(1)
+              }
+            }
+
+            let indexPath = Path.join('index')
+            if (nameLength === 1) {
+                indexPath = Path.join(indexPath, '1')
+            } else if (nameLength === 2) {
+                indexPath = Path.join(indexPath, '2')
+            } else if (nameLength === 3) {
+                indexPath = Path.join(indexPath, '3', buildpackInfo.name.slice(0, 2))
+            } else if (nameLength > 3) {
+                indexPath = Path.join(indexPath, buildpackInfo.name.slice(0, 2), buildpackInfo.name.slice(2, 4))
+            } else {
+                console.error(`buildpack name cannot be empty`)
+                process.exit(1)
+            }
+
+            indexPath = Path.join(indexPath, `${buildpackInfo.ns}_${buildpackInfo.name}`)
+            try {
+              const {data} = await github.repos.getContent({
+                owner: process.env.GITHUB_OWNER,
+                path: indexPath,
+                repo: 'buildpack-registry'
+              })
+
+              let buff = new Buffer.from(data.content, 'base64')
+              let fileContent = buff.toString('utf-8').trimEnd()
+              fileContent = fileContent + "\n" + JSON.stringify(buildpackInfo)
+
+              await createOrUpdateFileContents(indexPath, context.payload.issue.title, toBase64(fileContent), data.sha)
+
+              } catch (error) {
+                if (error.status && error.status === 404) {
+                  await createOrUpdateFileContents(indexPath, context.payload.issue.title, toBase64(JSON.stringify(buildpackInfo)))
+                } else {
+                  console.error(error)
+                  process.exit(1)
+                }
+              }
+
+      - uses: actions/github-script@v2
+        name: Close Issue on Success
+        with:
+          github-token: ${{ secrets.BOT_TOKEN }}
+          script: |
+            try {
+              await github.issues.createLabel({
+                owner: process.env.GITHUB_OWNER,
+                repo: 'buildpack-registry',
+                name: 'succeeded',
+                color: '008000'
+              });
+            } catch (error) {
+              if (!error.errors.find(err => err.code === 'already_exists')) {
+                console.error(error)
+                process.exit(1)
+              }
+            }
+            await github.issues.addLabels({
+              owner: process.env.GITHUB_OWNER,
+              repo: 'buildpack-registry',
+              issue_number: context.payload.issue.number,
+              labels: ['succeeded']
+            });
+            await github.issues.update({
+              owner: process.env.GITHUB_OWNER,
+              repo: 'buildpack-registry',
+              issue_number: context.payload.issue.number,
+              state: 'closed'
+            })
```

Comment:
Can you combine these into one call? The API docs seems to say you can have a labels array when updating an issue. https://developer.github.com/v3/issues/#update-an-issue